frontend/src/scenes/session-recordings/player/Seekbar.scss

```diff
@@ -113,8 +113,8 @@ $tick_hover_size: 5px;
                 border-radius: 50%;
                 right: -1 * floor($tick_size / 2) + floor($tick_hover_size / 2);
                 top: -1 * floor($tick_size / 2);
-                border: 0.5px solid $recording_player_container_bg;
-                background-color: $bg_light;
+                border: 0.5px solid $bg_light;
+                background-color: $recording_player_container_bg;
```

Comment:
This looks kind of odd when events are clumped closely
![image](https://user-images.githubusercontent.com/4813045/141203882-de83d277-bd05-4797-8858-b39447ad641b.png)

Reply:
Agreed, I think a darker border on lighter bg would make individual ticks stand out more. @clarkus What do you think?